Sources/AppStoreConnectCLI/Commands/Certificates/ReadDownloadCertificateCommand.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import ArgumentParser
+import Combine
+import Foundation
+import Files
+
+struct ReadDownloadCertificateCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+    commandName: "read",
+    abstract: "Get information about a certificate and download the certificate data.")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Argument(help: "The certificate’s serial number. (eg. 1A23BCDEF4G5D6C7)")
+    var serial: String
+
+    @Option(help: "The file download path and name. (eg. ./file.cer)")
+    var certificateOutput: String?
+
+    func run() throws {
+        let service = try makeService()
+
+        let certificate = try service
+            .readCertificate(serial: serial)
+            .await()
+
+        if let certificateOutput = certificateOutput {
+            guard let content = certificate.content,
+                let data = Data(base64Encoded: content) else {
+                throw CertificatesError.noContent
+            }
+
+            let (folderName, fileName) = File.folderAndFilename(from: certificateOutput)
+
+            let file = try Folder(path: folderName)
```

Comment:
```suggestion
            let file = try Folder(path: standardizedPath.deletingLastPathComponent)
```

Reply:
My only concern with the suggestion is to understand what `Folder(path:)` does when you pass an empty string.